src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -700,7 +703,12 @@ private boolean plantSapling(@NotNull final BlockPos location)
 
         worker.getCitizenStatusHandler().setLatestStatus(new TranslationTextComponent("com.minecolonies.coremod.status.planting"));
 
-        final int saplingSlot = findSaplingSlot();
+        int saplingSlot = findSaplingSlot();
+        if (saplingSlot == -1 && job.getTree().getSapling().getItem().isIn(fungi))
+        {
+            InventoryUtils.transferItemStackIntoNextBestSlotInItemHandler(job.getTree().getSapling(), worker.getInventoryCitizen());
+            saplingSlot = findSaplingSlot();
+        }
```

Comment:
This all seems weird, we're giving the lumberjck a free fungi if we can't find one?

Reply:
I may have misunderstood you yesterday. I thought he should be given one out of the blue. I can revert this if this is not intended.

Reply:
Yeah, we don't need this. In the next step (part (c) of what I had written you) when we take down the nether wart, we can add a random chance to get a shroom instead of the wart.

Reply:
Yeah, probably the more elegant solution to this! I'll revert the changes.